src/satellite/test.js

```diff
@@ -1016,7 +1016,6 @@ describe('Elastic()', () => {
     beforeEach(() => {
       client = Elastic();
       mock = client.mock;
-      mock.clearAll();
```

Comment:
What I was trying to say was, I think it was fine the way it was before. Was there really a need to change it to this way?